### ISSUE TITLE
cntk grammar interferes with sql grammar

### ISSUE THREAD

**Issue by aeschli** [Microsoft/vscode-tools-for-ai#21]
From https://github.com/Microsoft/vscode/issues/44016#issuecomment-366968234

When The Visual Studio Code Tools for AI is installed, SQL highlighting in VSCode changes.

The problem is that there's a clash of scope name:
`"scopeName": "source.sql"`

I agree that this should be handled better by VSCode, but it would good if you could change the scopes to something like source.cntk.

**Comment by alexandrudima** [comment]
To reproduce:

* `package.json`:

```json
{
	"publisher": "alex",
	"name": "44095",
	"version": "0.0.0",
	"engines": {
		"vscode": "^1.0.0"
	},
	"contributes": {
		"languages": [
			{
				"id": "cntk",
				"aliases": [
					"CNTK",
					"cntk"
				],
				"extensions": [
					".cntk",
					".bs"
				],
				"configuration": "./cntk.configuration.json"
			}
		],
		"grammars": [
			{
				"language": "cntk",
				"scopeName": "source.sql",
				"path": "./syntaxes/cntk.tmLanguage"
			}
		]
	}
}
```

**Comment by qfyin** [comment]
Thanks @aeschli for the suggestion.
We'll fix this very soon.
Please stay tuned.

**Comment by qfyin** [comment]
Fixed in 0.1.8
